### DISCUSSION TITLE
Flutter package for pulsar

### DISCUSSION THREAD

**Discussion by soumyaDghosh** [pulsar-edit/.github#132] [Community Packages]
I am starting to learn flutter, but in Pulsar there is no plugin for flutter. How am I supposed to test it? Can anyone help me?

**Comment by confused-Techie** [comment]
While I'm not familiar with Flutter, I'd assume you could follow their recommended steps to [get started](https://docs.flutter.dev/get-started/install) then use Pulsar to to write the actual code itself, and using their SDK or an Android Emulator for the actual testing.

Otherwise if any community members would like to create a package that aids in the flutter workflow then you'd be able to utilize that.

For anyone else looking into possibilities on this answer, it seems at one time there was a community package aimed at helping Flutter developers called [`flutter`](https://web.pulsar-edit.dev/packages/flutter) but unfortunately the maintainer has since deleted the repo.
